tests/translator/output/function_with_event_source_mapping.json

```diff
@@ -61,30 +64,35 @@
         ],
         "Policies": [
           {
-            "PolicyName": "MyFunctionForBatchingExampleRolePolicy0",
+            "PolicyName": "SQSPublishPolicyMyFunctionForBatchingExampleDynamoDBStreamEvent",
```

Comment:
`SQSPublishPolicyMyFunctionForBatchingExampleDynamoDBStreamEvent` will be changed as `MyFunctionForBatchingExampleDynamoDBStreamEventSQSPublishRolePolicy`